docs/configuration/README.md

```diff
@@ -31,3 +31,34 @@ var chartDifferentHoverMode = new Chart(ctx, {
     }
 });
 ```
+
+## Dataset Configuration
+
+Options may be configured direction on the dataset. Dataset defaults also allow for changing options globally across a dataset type avoiding the need to specify options for each dataset instance.
```

Comment:
I'm not sure what that first sentence means. 
Instead `direction on the dataset`, how about `at dataset level` or `by dataset type`?
Also `Dataset default also allow...`: it s not the defauts that allow anythin. How about `These options...` or `This configuration level...`?

Reply:
I think I meant "directly" instead of "direction". I've updated it the second sentence as well. Let me know if you think it sounds better now